### PR TITLE
Added docs of the core Claimant Model

### PR DESCRIPTION
**Pull request by mhutchinson**
This will be followed up by the Claimant Model: Logs doc if we're happy with the content, presentation, and location.

### CODE REVIEW

File: docs/claimantmodel/CoreModel.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+This paper introduces an intuitive, standalone model for a common scenario where one party needs to perform an action which is reasonable only if a claim made by another party is true. The Claimant Model allows formally verifiable constructs to be bound to a messy reality.
+
+## Model
+
+*This model describes roles, not actors. In a concrete deployment of this model, the same actor may play multiple roles, or multiple actors may need to work together to cover a role.*
+
+There is a **Claimant** that makes a **Claim** that is relied upon by a **Believer** as a precondition to take an action they would not have taken if the claim was false. Claims are represented by signed **Statements**. The veracity of a Claim can be verified by a **Claim Verifier**, who will notify a **Claim Arbiter** of any false Claims.
+
+## Example: Certification Claims
+* Claim<sup>CERT</sup>: (I, ${CA}, am authorized to certify $pubKey for $domain)
```
Comment (AlCutter) [inline docs/claimantmodel/CoreModel.md:12]: Could try using MD definitions or a table here to tidy?

File: docs/claimantmodel/CoreModel.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+<!-- TODO(mhutchinson): Using Logs to provide this discoverability is described in [Claimant Model: Logs](Logs.md). -->
+<!-- TODO(mhutchinson): Discuss Closed Loop Systems below and link to this. -->
+
+# Claim Requirements
+* The Statement for any claim must be signed by the Claimant to ensure that it is immutable and non-repudiable.
+* It must be possible to generate the Claim from the Statement without any additional context.
+* Claims must be falsifiable.
+  
+## Compound Claims
+When the Claim is composed of multiple subclaims, the Verifier role must be covered by actors that can verify all components of the Claim. For example, imagine a Claimant<sup>PRIME</sup> that provides Believer<sup>PRIME</sup> with an integer covered by Claim<sup>PRIME</sup>: "This is prime, and is uniquely issued to $believerID". This is a compound Claim: verification would need to confirm 1) the primality of the integer, and 2) that no other Believer had been issued the same value.
```
Comment (AlCutter) [inline docs/claimantmodel/CoreModel.md:32]: Might be worth a double \n\n at the `For example, ...`?

Comment (AlCutter) [inline docs/claimantmodel/CoreModel.md:32]: Could italicise the quoted `"This is prime..."`?

File: docs/claimantmodel/CoreModel.md (hunk 1 above)
Comment (mhutchinson) [reply docs/claimantmodel/CoreModel.md]: I've used a definition list. I'm not totally convinced it's better but I like compactness.

**Pull request merged**